_headers

```diff
@@ -1,2 +1,4 @@
 /search.json
   Access-Control-Allow-Origin: *
+/search/index.js
```

Comment:
Why not just `/search.js` that'd correspond to `/search.json`?

Reply:
OK, you convinced me. 😄